### PR TITLE
ref(feedback/issues): refactor linked comment

### PR DESCRIPTION
**Pull request by michellewzhang**
- fix a bug in which i converted the comment into a tuple, but it should be a string
- refactor the linked comment string to be more readable
- also change some instances from `Sentry issue` -> `Sentry Issue` for consistency

### CODE REVIEW

File: src/sentry/integrations/github/issues.py [hunk 1]
```diff
@@ -208,9 +208,22 @@ def get_link_issue_config(self, group: Group, **kwargs: Any) -> list[dict[str, A
...
+            url = group.get_absolute_url(params={"referrer": "github_integration"})
+            issue_id = group.qualified_short_id
+
+            return (
+                "{prefix}: [{issue_id}]({url})".format(
+                    prefix=prefix,
+                    url=absolute_uri(url),
+                    issue_id=issue_id,
+                ),
+            )
```
Comment (JoshFerge) [inline src/sentry/integrations/github/issues.py:226]:
```suggestion
            return f"{prefix}: [{issue_id}]({absolute_uri(url)})"
```

File: src/sentry/integrations/github/issues.py [hunk 2]
```diff
@@ -208,9 +208,22 @@ def get_link_issue_config(self, group: Group, **kwargs: Any) -> list[dict[str, A
...
-                return "Sentry feedback"
+                return "Sentry Feedback"
             else:
-                return "Sentry issue"
+                return "Sentry Issue"
+
+        def get_default_comment(group: Group) -> str:
+            prefix = get_linked_issue_comment_prefix(group)
+            url = group.get_absolute_url(params={"referrer": "github_integration"})
+            issue_id = group.qualified_short_id
```
Comment (JoshFerge) [inline src/sentry/integrations/github/issues.py:218]:
```suggestion
            issue_short_id = group.qualified_short_id
```

**Review by JoshFerge — APPROVED** [review]
great refactor! thank you!

**Comment by JoshFerge** [comment]
> fix a bug in which i converted the comment into a tuple, but it should be a string

should we add a regression test for this?

**Comment by michellewzhang** [comment]
> > fix a bug in which i converted the comment into a tuple, but it should be a string
> 
> should we add a regression test for this?

yes for sure, can address in a followup

**Pull request merged**